sci-libs/tensorflow-estimator/tensorflow-estimator-2.3.0.ebuild

```diff
@@ -8,15 +8,14 @@ MY_PN="estimator"
 MY_PV=${PV/_rc/-rc}
 MY_P=${MY_PN}-${MY_PV}
 
-inherit bazel distutils-r1 flag-o-matic toolchain-funcs
+inherit bazel distutils-r1
 
-DESCRIPTION="A high-level TensorFlow API that greatly simplifies machine learning programming"
+DESCRIPTION="High-level TensorFlow API that greatly simplifies machine learning programming"
 HOMEPAGE="https://www.tensorflow.org/"
 
 LICENSE="Apache-2.0"
 SLOT="0"
 KEYWORDS="~amd64"
-IUSE=""
 
 bazel_external_uris="
```

Comment:
$ pkgcheck scan --net -c SrcUriCheck  sci-libs/tensorflow-estimator
sci-libs/tensorflow-estimator
  TarballAvailable:

Reply:
thanks for the hint. 
However, do i need to make an `r1` if i change the source files? Because in that case i think it's not worth for the older ebuild.

Reply:
> However, do i need to make an `r1` if i change the source files? Because in that case i think it's not worth for the older ebuild.

Only if this change causes a change in things that are installed to the system (which is most likely not the case here)